### DISCUSSION TITLE
Always getting intendation error for my custom function while running on Python

### DISCUSSION THREAD

**Discussion by NikhileshAS** [brpapa/recursion-tree-visualizer#18] [Q&A]
<img width="431" alt="Screenshot 2024-04-12 at 12 31 09 PM" src="https://github.com/brpapa/recursion-tree-visualizer/assets/32016285/ba1f7b42-c429-4c99-b012-4c318f482090">
I always get this error.. the same code works in my local and other online compiler platforms.. Can anyone please point me out the error here?

**Comment by brpapa** [comment]
All code inserted below fn is interpreted as part of the fn function body. In this case, `def cal(...` should be indented.
